.github/workflows/package-react.yml

```diff
@@ -5,39 +5,37 @@ name: Package React
 on:
   push:
     branches:
-    - main
+      - main
     paths:
-    - 'ui/**'
-    - '!**.md'
+      - "ui/**"
+      - "!**.md"
   pull_request:
     paths:
-    - 'ui/**'
-    - '!**.md'
+      - "ui/**"
+      - "!**.md"
 
 defaults:
   run:
     working-directory: ./ui
 
 jobs:
-
   build-react:
-
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [10.x, 12.x, 14.x]
+        node-version: [12.x, 14.x]
     env:
-      CI: true
```

Comment:
I don't know what env.CI does here. Please advise.

Reply:
this is running the warnings as errors and previously blocking builds